lib/cryptoexchange/exchanges/liqnet/services/id_fetcher.rb

```diff
@@ -0,0 +1,20 @@
+module Cryptoexchange::Exchanges
+  module Liqnet
+    module Services
+      class IdFetcher
+        PAIR_SYMBOLS_URL = "#{Cryptoexchange::Exchanges::Liqnet::Market::API_URL}/markets"
+        PAIR_SYMBOLS = []
+        
+        def self.get_id(market_pair)
+          if PAIR_SYMBOLS.empty?
+           pairs_response = HTTP.get(PAIR_SYMBOLS_URL)
+           PAIR_SYMBOLS << pairs_response.parse(:json)
```

Comment:
this will cause issue if the constant is not being cleared (or assign a new client), perhaps dont 'cache' this for now since this is only required by OrderBooks and put a TODO comment.

Reply:
ok got it, will fix it to not store for now + add the comment